sru/modules.py

```diff
@@ -659,3 +600,42 @@ def make_backward_compatible(self):
             self.input_to_hidden = None
             for cell in self.rnn_lst:
                 cell.custom_m = None
+
+# some test code
+if __name__ == "__main__":
```

Comment:
ditto for this: can we move this into a test script in `tests/sru` please?

Reply:
+1